nucliadb/nucliadb/ingest/tests/integration/orm/test_orm_knowledgebox.py

```diff
@@ -49,35 +49,11 @@ def test_chunker():
     assert iterations == 0
 
 
-@pytest.fixture(scope="function")
-def tikv_driver_configured(tikv_driver):
-    from nucliadb.ingest.settings import settings
-    from nucliadb_utils.store import MAIN
-
-    prev_driver = settings.driver
-    settings.driver = "tikv"
-    settings.driver_tikv_url = tikv_driver.url
-    MAIN["driver"] = tikv_driver_configured
-
-    yield
-
-    settings.driver = prev_driver
-    MAIN.pop("driver", None)
-
-
-@pytest.fixture(scope="function")
-async def tikv_txn(tikv_driver):
-    txn = await tikv_driver.begin()
-    yield txn
-    await txn.abort()
-
-
 @pytest.mark.asyncio
 async def test_knowledgebox_delete_all_kb_keys(
```

Comment:
to make sure this test is passing with the tikv driver as before, shouldn't you add a local fixture overwriting `main_driver`?

Reply:
No, because this test is using `tikv_driver` directy. We can refactor it to use `get_driver` instead though